src/GovUk.Education.ExploreEducationStatistics.Common/Validators/ErrorDetails/FormatErrorDetail.cs

```diff
@@ -0,0 +1,11 @@
+#nullable enable
+namespace GovUk.Education.ExploreEducationStatistics.Common.Validators.ErrorDetails;
+
+/// <summary>
+/// Provides details about a value that was incorrectly formatted.
+/// </summary>
+/// <param name="Value">The invalid value</param>
+/// <param name="ExpectedFormat">The format that was expected</param>
+public record FormatErrorDetail(string Value, string ExpectedFormat)
+    : InvalidErrorDetail<string>(Value);
```

Comment:
I might be wrong, but I think you might end up with multiple copies of `Value` here...

https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/primary-constructors#:~:text=There%27s%20one%20potential%20concern%20with%20class%20hierarchies%20and%20primary%20constructors%3A%20it%27s%20possible%20to%20create%20multiple%20copies%20of%20a%20primary%20constructor%20parameter%20as%20it%27s%20used%20in%20both%20derived%20and%20base%20classes.%20The%20following%20code%20example%20creates%20two%20copies%20each%20of%20the%20owner%20and%20accountID%20field%3A

Reply:
Not sure there's a problem here:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/7cd33221-80e1-4dce-946c-ec26f75cd0af)

There's no shadowed `Value` when debugging, which looks fine to me!

Reply:
Discussed:

I believe the issue only arises when the primary constructor properties in the derived class are being used directly. The derived class then allocates new memory for them and they can become out of sync with the base-class instances.

Currently, we aren't introducing this issue so it is OK to leave this as-is.